src/coreclr/debug/createdump/createdumpwindows.cpp

```diff
@@ -63,9 +63,10 @@ CreateDump(const CreateDumpOptions& options)
         printf_error("Invalid dump path '%s' - %s\n", dumpPath.c_str(), GetLastErrorString().c_str());
         goto exit;
     }
-
+    
+    int retryCount = 5;
```

Comment:
Maybe we should change the retry count to 10 like dotnet-dump collect does now.

Reply:
done